docusaurus.config.js

```diff
@@ -44,8 +44,315 @@ const config = {
         redirects: [
           {
             to: '/docs/guides/implementation-guide',
-            from: '/docs/protocol/implementation-guide',
+            from: [
+              '/docs/protocol/spec',
+              '/docs/protocol/implementation-guide'
+            ],
+          },
+          {
+            to: '/docs/reference/messaging/messaging-interface',
+            from: [
+              '/docs/apis-and-sdks/',
+              '/docs/apis-and-sdks/messaging-api',
+            ],
+          },
+          {
+            to: 'https://v2.hyperlane.xyz/docs/apis-and-sdks/accounts',
+            from: ['/docs/apis-and-sdks/accounts'],
+          },
+          {
+            to: 'https://v2.hyperlane.xyz/docs/apis-and-sdks/query',
+            from: ['/docs/apis-and-sdks/query'],
+          },
+          {
+            to: 'https://v2.hyperlane.xyz/docs/build-with-hyperlane/quickstarts/accounts',
+            from: ['/docs/build-with-hyperlane/quickstarts/accounts']
+          },
+          {
+            to: 'https://v2.hyperlane.xyz/docs/build-with-hyperlane/quickstarts/queries',
+            from: ['/docs/build-with-hyperlane/quickstarts/queries']
+          },
+          {
+            to: '/docs/reference/applications/warp-routes',
+            from: [
+              '/docs/apis-and-sdks/warp-api',
+              '/docs/build-with-hyperlane/examples',
+              '/docs/build-with-hyperlane/quickstarts/hyperlane-quickstart',
+              '/docs/apis-and-sdks/building-applications',
+              '/docs/apis-and-sdks/building-applications/example-usage',
+              '/docs/apis-and-sdks/building-applications/nodejs-sdk',
```

Comment:
Could link instead to the typescript-sdk in monorepo, but then you lose consistency with other */nodejs-sdk* links